README.md

```diff
@@ -7,10 +7,14 @@ WebDriver.
 
 ## Configure your Bazel project
 
-For all languages, you need to add the following to your MODULE.bazel file:
+For each language , you need to add the following to your MODULE.bazel file:
 
 ```bzl
-bazel_dep(name = "rules_webtesting", version = "0.4.0")
+bazel_dep(name = "rules_webtesting_${language}", version = "0.4.0")
+```
+For example for Java:
+```bzl
+bazel_dep(name = "rules_webtesting_java", version = "0.4.0")
```

Comment:
Don't they still need to add "rules_webtesting" to refer to be browsers?

Reply:
For all languages, you need to add the following to your MODULE.bazel file:
```bzl
bazel_dep(name = "rules_webtesting", version = "0.4.0")
```

For each language that is used by your project, you need to add the corresponding sub module:

```bzl
bazel_dep(name = "rules_webtesting_${language}", version = "0.4.0")
```

For example for Java:
```bzl
bazel_dep(name = "rules_webtesting", version = "0.4.0")
bazel_dep(name = "rules_webtesting_java", version = "0.4.0")
```